dozer/__main__.py

```diff
@@ -17,10 +17,6 @@
     'tba': {
         'key': 'Put TBA API key here'
     },
-    'toa': {
```

Comment:
An FTCScout API key section should be added if necessary.

Reply:
As of right now we don’t require an API key.